equinix/resource_metal_reserved_ip_block.go

```diff
@@ -311,7 +311,7 @@ func resourceMetalReservedIPBlockCreate(d *schema.ResourceData, meta interface{}
 }
 
 func resourceMetalReservedIPBlockUpdate(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*packngo.Client)
+	client := meta.(*Config).metal
```

Comment:
Verified that this is the only place where this was missed. This was the format used in terraform-provider-metal and this was missed when backporting.